views/index.hbs

```diff
@@ -1,3 +1,19 @@
-<div>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+    <title>Home</title>
+</head>
+<body>
+    <div>
+        <img src="/public/images/beer.png" class="rounded mx-auto d-block" alt="pint">
+        <button href="/beers" type="button" class="btn btn-primary">Check the beers!</button>
```

Comment:
You can only use the "href" attribute on `<a>` or `<link>`.

The correct way to write line 12 is 
```
<a href="/beers">
  <button  type="button" class="btn btn-primary">Check the beers!</button>
</a>